airootfs/usr/share/vx-img/flash-image.sh

```diff
@@ -71,6 +71,25 @@ fi
 
 clear
 
+# Detect if this disk already has VotingWorks data on it and copy the machine config
+vg=$(vgscan | sed -s 's/.*"\(.*\)".*/\1/g')
```

Comment:
I'm having trouble understanding how this tells us there is /vx/config data on the system. Is this just looking at whether there are logical volumes and from that assuming we've already installed?

Reply:
so this comment applies to the next ~11 lines of the script, not just this one. Basically the algorithm is this:
1. Check if there is an LVM volume group on the device
2. If there is, check that there is a `var` partition (due to our setup this also includes the symlinked `/vx/config` folder)
3. Check that `/vx/config` exists, if it does tar it up. 

It's probably not a perfect algorithm. If we wanted to key on only the specific volume group name (typically `Vx--vg`) we could do that to be more specific. I could also probably be checking that `/vx/config` is configured correctly as a symlink to `/var/vx/config`, and that it has contents...

Reply:
oh got it. I don't know why I was hung up on that first line. OK this makes sense, I think it's a fine algorithm!